Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: test
+
+CONTAINER := $(shell docker run --rm --add-host="posttest.localhost:127.0.0.1" -e "RUBYOPT=-W0" -d fake-s3 rake test_server)
```

Comment:
`RUBYOPT=-W0` was required to remove all the `/usr/lib/ruby/gems/2.3.0/gems/right_http_connection-1.5.0/lib/base/net_fix.rb:51:in `rbuf_fill': Object#timeout is deprecated, use Timeout.timeout instead.` warnings from the test output.